docs/devguide/README.md

```diff
@@ -70,8 +68,6 @@ Click next, and browse into the `jmc/core` folder. Select all the core projects
 
 Next select _File | Import…_ and select _Maven / Existing Maven Project_ again, but this time from the root.
 
-Next we will import the project which contains the launchers. Select _File | Import…_ and then select _Existing Projects into Workspace_. Find the `configuration/ide/eclipse` folder and click Ok.
-
```

Comment:
Why is the step to import the launchers removed?

Reply:
I wrote the reason in the specific commit: Eclipse auto import them when in a previous step, doing this actually made the launchers appear twice in the menu, now they only appear once.